ProjectManagement/Startup.cs

```diff
@@ -59,7 +59,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IUserManager, ProjectManagement.BusinessLogic.Services.Implementation.UserMananger>(x =>
                         new UserMananger(
                             x.GetRequiredService<IHttpContextAccessor>(), x.GetRequiredService<IUserRepository>()));
-            services.AddScoped<IUserService, UserService>();
+            services.AddScoped<IFileService, FileService>();
+            services.AddScoped<IUserService, UserService>(x=>new UserService(x.GetRequiredService<IUserRepository>(),x.GetRequiredService<IFileService>()));
```

Comment:
Why do you need to use a delegate instantiation here?

Reply:
I used it as ways of passing parameters to constructor. But now I see that this is not necessary.